packages/story-editor/src/app/rightClickMenu/menus/foregroundMediaMenu.js

```diff
@@ -203,6 +210,13 @@ function ForegroundMediaMenu({ parentMenuRef }) {
       <ContextMenuComponents.MenuButton onClick={handleOpenScaleAndCrop}>
         {scaleLabel}
       </ContextMenuComponents.MenuButton>
+
+      {cropHidden && isOffCanvas() && (
+        <ContextMenuComponents.MenuButton onClick={handleCropHidden}>
+          {RIGHT_CLICK_MENU_LABELS.CROP_HIDDEN}
+        </ContextMenuComponents.MenuButton>
+      )}
```

Comment:
We should only display this for media elements that can be cropped

(Aside: super nice that this works for images too!)

Reply:
Added isVideo check.